dingo/PolytopeSampler.py

```diff
@@ -307,7 +307,7 @@ def sample_from_fva_output(
         Keyword arguments:
         min_fluxes -- minimum values of the fluxes, i.e., a n-dimensional vector
         max_fluxes -- maximum values for the fluxes, i.e., a n-dimensional vector
-        biomass_function -- the biomass objective function
+        objective_function -- the biomass objective function
```

Comment:
should you also change the comment?

Reply:
Yes, thank you. Added a commit with the changes.